src/feeds.js

```diff
@@ -308,64 +306,86 @@ feeds.fetchEventsFromCalendar_ = function(feed, callback) {
     }
     _gaq.push(['_trackEvent', 'Fetch', 'Events']);
```

Comment:
Since we will now do the fetching logic multiple times, moving the `getAuthToken` up to do it only once per `fetchEventsFromCalendar_` call.